src/components/layouts/sidebar.tsx

```diff
@@ -121,6 +120,45 @@ const Sidebar = (): JSX.Element => {
 							<span className="ml-3">{t("networks")}</span>
 						</Link>
 					</li>
+					{me?.memberOfOrgs?.length > 0 ? (
+						<>
+							<li className="my-px">
+								<span className="my-4 flex px-4 text-sm font-medium uppercase text-primary">
+									Organiszation
```

Comment:
Did you mean "Organization"? :)

Reply:
Yes, but these hardcoded text is just kinda placeholders. All text will be added to the translation files later with correct spelling. :)